[2022-09-02T00:03:56.145-0400] {plugins_manager.py:292} DEBUG - Plugins are already loaded. Skipping.
[2022-09-02T00:03:56.153-0400] {__init__.py:50} DEBUG - Loading core task runner: StandardTaskRunner
[2022-09-02T00:03:56.162-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:56.166-0400] {base_task_runner.py:70} DEBUG - Planning to run as the  user
[2022-09-02T00:03:56.170-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> from DB
[2022-09-02T00:03:56.179-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2022-09-02T00:03:56.179-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-02T00:03:56.185-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2022-09-02T00:03:56.185-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-02T00:03:56.185-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]>
[2022-09-02T00:03:56.186-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-02T00:03:56.190-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [queued]>
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-02T00:03:56.195-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-02T00:03:56.209-0400] {taskinstance.py:1389} INFO - Executing <Task(_PythonDecoratedOperator): generate_report> on 2022-09-02 04:03:42.436351+00:00
[2022-09-02T00:03:56.211-0400] {standard_task_runner.py:52} INFO - Started process 20300 to run task
[2022-09-02T00:03:56.218-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_etl', 'generate_report', 'manual__2022-09-02T04:03:42.436351+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmpyh8vk_ru', '--error-file', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmp0k3i76zz']
[2022-09-02T00:03:56.220-0400] {standard_task_runner.py:80} INFO - Job 36: Subtask generate_report
[2022-09-02T00:03:56.226-0400] {cli_action_loggers.py:66} DEBUG - Calling callbacks: [<function default_action_log at 0x7fbc2ab283a0>]
[2022-09-02T00:03:56.242-0400] {settings.py:398} DEBUG - Disposing DB connection pool (PID 20300)
[2022-09-02T00:03:56.243-0400] {settings.py:260} DEBUG - Setting up DB connection pool (PID 20300)
[2022-09-02T00:03:56.243-0400] {settings.py:330} DEBUG - settings.prepare_engine_args(): Using NullPool
[2022-09-02T00:03:56.286-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:56.302-0400] {task_command.py:371} INFO - Running <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [running]> on host FAS-EALC-LIU-2
[2022-09-02T00:03:56.304-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:56.335-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-02T00:03:56.342-0400] {taskinstance.py:943} DEBUG - Clearing XCom data
[2022-09-02T00:03:56.543-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_etl
AIRFLOW_CTX_TASK_ID=generate_report
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T04:03:42.436351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-02T04:03:42.436351+00:00
[2022-09-02T00:03:56.544-0400] {__init__.py:146} DEBUG - Preparing lineage inlets and outlets
[2022-09-02T00:03:56.544-0400] {__init__.py:190} DEBUG - inlets: [], outlets: []
[2022-09-02T00:03:56.545-0400] {logging_mixin.py:115} INFO - ('126.42', '126.41')
[2022-09-02T00:03:56.546-0400] {logging_mixin.py:115} INFO - Summary:
[2022-09-02T00:03:56.546-0400] {logging_mixin.py:115} INFO -  T. Rowe Price Group, Inc. is an American publicly owned global investment management firm that offers funds, advisory services, account management, and retirement plans and services for individuals, institutions, and financial intermediaries. The firm has assets under management of more than $1.6 trillion and annual revenues of $6.2 billion as of 2020, placing it 447 on the Fortune 500 list of the largest U.S. companies. Headquartered at 100 East Pratt Street in Baltimore, Maryland, it has 5,000 employees in Baltimore and 16 international offices serve clients in 47 countries around the world.The firm was founded in 1937 by Thomas Rowe Price, Jr. who is best known for developing the growth stock philosophy of investing. As of 2019, the company is focused on active management after strategically deciding against a major initiative in passive investment.Consistently ranked among the world's top asset managers, T. Rowe Price was named one of the best places to work in money management by Pensions&Investments and was one of Fortune's most admired companies in 2020. 
[2022-09-02T00:03:56.547-0400] {logging_mixin.py:115} INFO - Further reading:
[2022-09-02T00:03:56.547-0400] {logging_mixin.py:115} INFO -  https://en.wikipedia.org/wiki/T._Rowe_Price
[2022-09-02T00:03:56.548-0400] {python.py:173} INFO - Done. Returned value was: None
[2022-09-02T00:03:56.550-0400] {__init__.py:107} DEBUG - Lineage called with inlets: [], outlets: []
[2022-09-02T00:03:56.550-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:56.569-0400] {taskinstance.py:1418} DEBUG - Clearing next_method and next_kwargs.
[2022-09-02T00:03:56.582-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_etl, task_id=generate_report, execution_date=20220902T040342, start_date=20220902T040356, end_date=20220902T040356
[2022-09-02T00:03:56.583-0400] {taskinstance.py:2349} DEBUG - Task Duration set to 0.39611
[2022-09-02T00:03:56.613-0400] {cli_action_loggers.py:84} DEBUG - Calling callbacks: []
[2022-09-02T00:03:56.651-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-02T00:03:56.652-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.generate_report manual__2022-09-02T04:03:42.436351+00:00 [running]> from DB
[2022-09-02T00:03:56.674-0400] {dagrun.py:642} DEBUG - number of tis tasks for <DagRun test_etl @ 2022-09-02 04:03:42.436351+00:00: manual__2022-09-02T04:03:42.436351+00:00, state:running, queued_at: 2022-09-02 04:03:42.469689+00:00. externally triggered: True>: 0 task(s)
[2022-09-02T00:03:56.675-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
